lib/query.js

```diff
@@ -71,7 +81,10 @@ export const isPolynomialTerm = (node) => {
         return true
     } else if (isPow(node)) {
         const [base, exponent] = node.args
-        return (isIdentifier(base) || isPolynomial(base)) && isPolynomialTerm(exponent)
+        const positiveExp = isNumber(exponent)
+            ? getValue(exponent) >= 1
+            : true
```

Comment:
```
const positiveExp = isInteger(exponent) && getValue(exponent) > 0 || ...
```
We could leave the `...` or make it something simple like `isIdentifier(exponent)` for now.